packages/fx-core/src/common/spec-parser/manifestUpdater.ts

```diff
@@ -55,40 +88,51 @@ export async function generateCommands(
       if (pathItem) {
         const operations = pathItem;
 
-        // Currently only support GET method
-        const operationItem = operations.get;
-
-        if (operationItem) {
-          const parameters: Parameter[] = [];
-          const paramObject = operationItem.parameters;
-
-          if (paramObject) {
-            paramObject.forEach((param: OpenAPIV3.ParameterObject | OpenAPIV3.ReferenceObject) => {
-              param = param as OpenAPIV3.ParameterObject;
-              parameters.push({
-                name: param.name,
-                title: updateFirstLetter(param.name),
-                description: param.description ?? "",
-              });
-            });
-          }
+        // Currently only support GET and POST method
+        for (const method in operations) {
+          if (method === ConstantString.PostMethod || method === ConstantString.GetMethod) {
+            const operationItem = operations[method];
+            if (operationItem) {
+              const parameters: Parameter[] = [];
+              const paramObject = operationItem.parameters as OpenAPIV3.ParameterObject[];
+
+              if (paramObject) {
+                paramObject.forEach((param: OpenAPIV3.ParameterObject) => {
+                  parameters.push({
+                    name: param.name,
+                    title: updateFirstLetter(param.name),
+                    description: param.description ?? "",
+                  });
+                });
+              }
+
+              if (operationItem.requestBody) {
+                const requestJson = (operationItem.requestBody as OpenAPIV3.RequestBodyObject)
```

Comment:
Do we check content to be "application/json" when validating? or any requirements about content type? Just trying to think about possible values when we run generate()

Reply:
From Josh's validation rule, the request body should be "application/json", other type is not supported

Reply:
so we will have plan to update our validate() then?

Reply:
noticed that we've already have this in this PR. Resolved.